src/tests/ftest/recovery/container_cleanup.py

```diff
@@ -34,9 +34,9 @@ def test_container_label_inconsistency(self):
 
         Jira ID: DAOS-12289
 
-        :avocado: tags=all,pr
-        :avocado: tags=vm
-        :avocado: tags=recovery,container_cleanup
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
```

Comment:
If moving to HW, we should change the config to `storage: auto`
https://github.com/daos-stack/daos/blob/1d3929216fe5d1cbae325887c0e2487fec5ed514/src/tests/ftest/recovery/container_cleanup.yaml#L13-L16

Reply:
I will fix that.